.buildkite/pipeline.yml

```diff
@@ -21,3 +21,8 @@ steps:
 
   - label: Upload artifacts
     command: .buildkite/setup_and_upload_artifact.sh && docker image prune -f -a
+
+  - label: Sign Windows installer
+    command: .buildkite/sign_windows_installer.bat
+    agents:
+      queue: "windows-sign"
```

Comment:
Does this mean only agents with this tag is able to sign this? So vader won't start signing?

Reply:
>Have you tested if the produced signed installer works on windows and doesn't prompt with the 'unknown developer' dialog box?

Yeah I tested it, here the sample build [artifact link](https://buildkite.com/organizations/learningequality/pipelines/mrpau-windows-sign-testing/builds/54/jobs/6f41ee33-996f-4ccc-9900-29eeb4e23bf2/artifacts/790970f3-768f-4650-99d8-bb8c0e08b847) .

>Does this mean only agents with this tag is able to sign this? So vader won't start signing?

The Windows signing credentials is on the `Windows-2016` agent. If we sign it at vader we need to add those signing credentials there.